.github/workflows/verilator_compatibility.yml

```diff
@@ -0,0 +1,36 @@
+name: DaCe Verilator Compatibility Check
+on:
+  workflow_dispatch:
+    inputs:
+      reason:
+        description: 'Reason for the trigger'
+        required: true
+        default: 'Check compatibility'
+  schedule:
+    - cron: '0 0 1 * *' # monthly
```

Comment:
Please remove

Reply:
Sure, i can remove this. I thought it would be a good idea to test the compatibility monthly. As soon as nobody is working on the Verilator integration anymore, there will be no more manual triggers and we might loose compatibility at some point without detecting this.